plugins/janus_recordplay.c

```diff
@@ -615,6 +621,13 @@ static const char *janus_recordplay_parse_codec(const char *dir, const char *fil
 					return NULL;
 				}
 				const char *c = json_string_value(codec);
+				if (data) {
```

Comment:
Code style: no space between `if` and bracket.

Reply:
Corrected.